CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 ### Fixes
 
 * Fixes SASS warnings.
+* Fixes a bug where related documents were not exported if they were only in one (draft or published) mode.
```

Comment:
Sounds like this was a bit worse than we thought; should we ship a hotfix for this module? I don't think it's a big deal to ship one

Reply:
It's not exported in a particular mode is what it says. So `draft` doc exports only `draft` relation. It's actually bad, because `draft` relation can't be referenced.